src/apps/storefront/index.md

```diff
@@ -0,0 +1,28 @@
+# Storefront Overview
+
+Frontend (Storefront) is a crucial aspect of your e-commerce business that online customers see and interact with, making it possible for them to view products, place orders, and manage their purchases from their devices.
+
+[Shopware's Storefront is well structured](https://developer.shopware.com/docs/concepts/framework/architecture/storefront-concept). It predominantly consists of design templates that match your brand and store themes.
+
+## Build
+
+Shopware server builds the [B2B-Suite Storefront](https://developer.shopware.com/docs/products/extensions/b2b-suite/guides/storefront). However, you can modify the whole appearance of the Storefront by customizing storefront components such as [template files](https://developer.shopware.com/docs/guides/plugins/plugins/storefront/customize-templates), [Javascript sources](https://developer.shopware.com/docs/guides/plugins/plugins/storefront/add-custom-javascript), [page](https://developer.shopware.com/docs/guides/plugins/plugins/storefront/add-custom-page), [pagelets](https://developer.shopware.com/docs/guides/plugins/plugins/storefront/add-custom-pagelet), [filters](https://developer.shopware.com/docs/guides/plugins/plugins/storefront/add-custom-sorting-product-listing), [responsive themes](https://developer.shopware.com/docs/guides/plugins/themes), [styling](https://developer.shopware.com/docs/guides/plugins/plugins/storefront/add-custom-styling), [icons](https://developer.shopware.com/docs/guides/plugins/plugins/storefront/add-icons), etc.
```

Comment:
I think you should leave the B2B suite out of this

Reply:
I am yet to work on this article.